src/south/ProtocolHandler.class.js

```diff
@@ -483,6 +491,19 @@ class ProtocolHandler {
     return timestamp
   }
 
+  /**
+   * Replace the variables such as @CurrentDate in the file name with their values
+   * @param {string} filename - The filename to change
+   * @param {number} queryPart - The part of the query being executed
+   * @return {string} newFilename - The renamed file name
+   */
+  replaceFilenameWithVariable(filename, queryPart) {
```

Comment:
It is very similar to the overwritten version in HistoryQuery. Would not be better to create a `connectorName` South variable initiated with `this.dataSource.name` and overwrite it in HistoryQuery with `${this.dataSource.name}-${this.application.name}`? This way don't have to overwrite this method.